TallyBoxOutput.cpp

```diff
@@ -53,6 +53,14 @@ void setBrightnessSettingMode(tallyBoxOutput_t ch, bool enable)
   }
 }
 
+bool getBrightnessSettingMode(tallyBoxOutput_t& ch)
+{
+  bool ret = false;
+  ret = brightnessSettingModeEnabled;
+  ch = brightnessSettingModeChannel;
```

Comment:
return with value missing
```suggestion
  ch = brightnessSettingModeChannel;
  
  return ret;
```